tests/phpunit/CRM/Activity/BAO/ActivityTest.php

```diff
@@ -1655,36 +1655,57 @@ public function testSendEmailWillReplaceTokensUniquelyForEachContact() {
     }
   }
 
+  public function testSmartyEnabled() {
+    putenv('CIVICRM_MAIL_SMARTY=1');
+    $this->createLoggedInUser();
+    $contactID = $this->individualCreate(['last_name' => 'Red']);
+    CRM_Activity_BAO_Activity::sendEmail(
+      [
+        $contactID => [
+          'preferred_mail_format' => 'Both',
+          'contact_id' => $contactID,
+          'email' => 'a@example.com',
+        ],
+      ],
+      '{contact.first_name} {$contact.first_name}',
+      '{contact.first_name} {$contact.first_name}',
+     '{contact.first_name} {$contact.first_name}',
```

Comment:
phpcs doesn't catch these indents I guess. Also down on line 1732.

Reply:
That's not like phpcs - to be forgiving - fixed